roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -350,6 +342,40 @@ def add_epochs(imaging, nwbfile):
 
         return nwbfile
 
+    @staticmethod
```

Comment:
this method just overwrites some of the default fields like channel_names, imaging_rate, and image_size (for TwoPhotonSeries)